docs/advanced/porting_guide.md

```diff
@@ -1,16 +1,14 @@
 # Porting guide: adding target support to mbed OS 5
 
-There are two typical ways to add target support to mbed OS: Either you'll be adding a [completely new microcontroller](#adding-a-new-microcontroller) and board, or you'll be adding a new board that contains an [already-supported microcontroller](#adding-a-new-board-or-module). 
+Adding a new microcontroller to mbed OS depends on CMSIS-CORE and CMSIS-Pack. Please make sure that the microcontroller already has these available.
 
-There is a dependency on CMSIS-CORE and CMSIS-Pack, so make sure the microcontroller already has these available.
+## Adding a new microcontroller and Board
 
-## Adding a new microcontroller
-
-Add the empty target implementation reference to the working directory using: 
+First fork the mbed-os repository on github into your own user account. We will use the metasyntatic variable `USERNAME` to refer to your username in the following documentation, `MCU_NAME` to refer to the new microcontroller you are adding and `BOARD_NAME` to refer to the new board you are adding. Import an mbed OS example and make it use your fork of mbed-os using: 
 
 ```
-mbed import mbed-os-example-new-target
-cd mbed-os-example-new-target\mbed-os
+mbed import mbed-os-example-blinky
+cd mbed-os-example-blinky\mbed-os
 git checkout master
 git pull
 git checkout -b my-new-target
```

Comment:
I fixed it.

This comment would have collapsed if you had commented on the line you were referring to.